src/reducers/SearchSlice.jsx

```diff
@@ -12,12 +12,20 @@ const initialState = {
   searchResults: [],
   searchValue: '',
   loading: 'idle',
```

Comment:
It would be good to document the expected values and their meaning. In the profiler we're using a type system to help us with that.

It would be good to document which transitions are expected too (eg: idle -> pending, etc).

Reply:
This is an old component with very few comments here ahah.

Here is a simple example: https://github.com/firefox-devtools/profiler/blob/95d40cb654241bd0a6bc1b38dca8def5338b1a2a/src/types/actions.js#L131-L138

But the meaning might be obvious in this case so it's really up to you :) Try to put yourself in your own brain reading this in 6 months, and add the information you think could be useful to the future you. You know better than me :-)

Reply:
filed #75